src/ripple/app/main/Main.cpp

```diff
@@ -150,7 +150,7 @@ void printHelp (const po::options_description& desc)
            "     version\n"
            "     server_info\n"
            "     sign <private_key> <json> [offline]\n"
-           "     sign_for\n"
+           "     sign_for <signer_address> <signer_private_key> <json> [offline]\n"
            "     stop\n"
            "     submit <tx_blob>|[<private_key> <json>]\n"
            "     submit_multisigned\n"
```

Comment:
Okay.  I looked at the `--help` message as it stands today, and I didn't see any `tx_json`, which is why I didn't use it in the `sign_for` syntax hint.  But your suggestion is to replace `json` with `tx_json` in: `sign`, `sign_for`, `submit`, and `submit_multisigned`.  I can do that.